templates/cassiopeia/scss/vendor/joomla-custom-elements/joomla-alert.scss

```diff
@@ -9,13 +9,17 @@
   width: 100%;
   min-width: 16rem;
   padding: 0;
-  margin-bottom: 1rem;
+  margin-bottom: 0;
   color: var(--gray-dark);
   background-color: $white;
   border: 1px solid var(--alert-accent-color, transparent);
   border-radius: .25rem;
   transition: opacity .15s linear;
 
+  + joomla-alert {
```

Comment:
@hans2103 What does the "+" mean here?

Reply:
Hmm, I see.

Reply:
It combines two selectors having the same parent and the second one must come immediately after the first. A very sophisticated construction.

Reply:
Yes, that's why I haven't seen it so often.